src/plugins/home/server/tutorials/cisco_logs/index.ts

```diff
@@ -35,31 +35,32 @@ export function ciscoLogsSpecProvider(context: TutorialContext): TutorialSchema
   return {
     id: 'ciscoLogs',
     name: i18n.translate('home.tutorials.ciscoLogs.nameTitle', {
-      defaultMessage: 'Cisco',
+      defaultMessage: 'Cisco logs',
     }),
     moduleName,
     category: TutorialsCategory.SECURITY_SOLUTION,
     shortDescription: i18n.translate('home.tutorials.ciscoLogs.shortDescription', {
-      defaultMessage: 'Collect and parse logs received from Cisco ASA firewalls.',
+      defaultMessage: 'Collect Cisco network device logs over syslog or from a file.',
     }),
     longDescription: i18n.translate('home.tutorials.ciscoLogs.longDescription', {
       defaultMessage:
-        'This is a module for Cisco network device’s logs. Currently \
-supports the "asa" fileset for Cisco ASA firewall logs received over syslog or read from a file. \
+        'This is a module for Cisco network devices logs (ASA, FTD, IOS, Nexus). It includes the following filesets for receiving logs over syslog or read from a file: \
 [Learn more]({learnMoreLink}).',
       values: {
         learnMoreLink: '{config.docs.beats.filebeat}/filebeat-module-cisco.html',
       },
     }),
     euiIconType: '/plugins/home/assets/logos/cisco.svg',
     artifacts: {
-      dashboards: [],
-      application: {
-        path: '/app/security',
-        label: i18n.translate('home.tutorials.ciscoLogs.artifacts.dashboards.linkLabel', {
-          defaultMessage: 'Security App',
-        }),
-      },
+      dashboards: [
+        {
+          id: 'a555b160-4987-11e9-b8ce-ed898b5ef295',
+          linkLabel: i18n.translate('home.tutorials.ciscoLogs.artifacts.dashboards.linkLabel', {
+            defaultMessage: 'ASA Firewall Dashboard',
```

Comment:
It seems it refers not only to ASA (https://github.com/elastic/kibana/pull/77237/files#diff-11a1d7b8a383edfaa70f21c8c0a2eb71R47). Not sure if is worth changing this also

Reply:
Good observation. Each tutorial has space for a single dashboard or Kibana app link. So I had to choose to which one to include. The text refers to the other parts of the module. I should probably create separate tutorials for each dataset like we are going to do for the packages. I added a TODO for this to the issue for this task. WDYT about merging as is and do a follow up?